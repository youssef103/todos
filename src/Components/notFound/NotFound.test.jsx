import React from "react";
import { NotFound } from "./NotFound";
import { shallow } from "enzyme";

test("should render NotFound", () => {
  const wrapper = shallow(<NotFound />);

  expect(wrapper).toMatchSnapshot();
});
