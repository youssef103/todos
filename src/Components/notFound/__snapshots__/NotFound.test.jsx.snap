// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound 1`] = `
<div>
  NotFound
</div>
`;
